controllers/k8sgpt_controller.go

```diff
@@ -218,6 +223,52 @@ func (r *K8sGPTReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 					Namespace: k8sgptConfig.Namespace,
 				},
 			}
+			if k8sgptConfig.Spec.Backstage {
+				labelKey := "backstage.io/kubernetes-id"
```

Comment:
Make this a const in the `pkg/integrations/integrations.go`

Reply:
done